docs/2-Collection_Containers.adoc

```diff
@@ -965,6 +965,90 @@ They are named *{IntHashSet}*, *{FloatHashSet}* etc.
 *{BooleanHashSet}* is implemented using a single integer to hold one of four possible states: ([], [F], [T], or [T, F]).
 All other sets use open addressing and quadratic probing.
 
+.There are different ways you can create primitive sets, which are listed as below.
```

Comment:
"as below" -> "below"

Reply:
Thank you for suggestions, I have updated adoc with suggested changes.